src/quadpype/hosts/tvpaint/plugins/publish/extract_sequence.py

```diff
@@ -285,8 +297,7 @@ def render_review(
             output_dir (str): Directory where files will be stored.
             mark_in (int): Starting frame index from which export will begin.
             mark_out (int): On which frame index export will end.
-            scene_bg_color (list): Bg color set in scene. Result of george
-                script command `tv_background`.
+            apply_bg_back_command (str): George script command to put the BG back as set in current scene.
```

Comment:
Why passing the variable `apply_bg_back_command` since it's set by you in `self` before.